Formula/spr.rb

```diff
@@ -0,0 +1,58 @@
+class Spr < Formula
+  desc "Submit pull requests for individual, amendable, rebaseable commits to GitHub"
+  homepage "https://github.com/getcord/spr"
+  url "https://github.com/getcord/spr.git",
+    tag:      "v1.3.4",
+    revision: "1b05ef36f0beb0ff8bdb376bf98ce77a41e41605"
```

Comment:
Align the arguments of `url`, please, like so:
```suggestion
  url "https://github.com/getcord/spr.git",
      tag:      "v1.3.4",
      revision: "1b05ef36f0beb0ff8bdb376bf98ce77a41e41605"
```
Though, do we really need a Git clone? Can we use a tarball instead?

Reply:
I copied this from some other formula and didn't think much about it. But I guess it's cheaper to download a tarball than clone a Git repo.

Changing this.